app/controllers/v1/docs_controller.rb

```diff
@@ -2,9 +2,11 @@
 
 module V1
   # DocsController
+  # rubocop:disable Rails/ApplicationController
   class DocsController < ActionController::Base
     def index
       render layout: false
     end
   end
+  # rubocop:enable Rails/ApplicationController
```

Comment:
Probably not worth changing, but I've had a lot of luck with putting the disable line on the end of the line for the block you want to disable it on and it become re-enabled automatically at the end of the block.  Probably not worth changing.  Just an observation.